52_wa_diff_shipment.py

```diff
@@ -0,0 +1,263 @@
+# importing the required libraries
+import pandas as pd
+import numpy as np
+import altair as alt
+import statsmodels.api as sm
+import statsmodels.formula.api as smf
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+# loading data for drug prescriptions in Florida and neighbouring states
+prescriptions = pd.read_csv("05_cleaned_data/prescriptions_wa.csv", low_memory=False)
+
+# reducing the number of columns in the drug prescriptions dataset by including only the relevant attributes to create a new dataset
+
+prescriptions_reduced = prescriptions[
+    [
+        "DRUG_CODE",
+        "DRUG_NAME",
+        "QUANTITY",
+        "UNIT",
+        "STRENGTH",
+        "CALC_BASE_WT_IN_GM",
+        "DOSAGE_UNIT",
+        "Product_Name",
+        "Ingredient_Name",
+        "Measure",
+        "MME_Conversion_Factor",
+        "dos_str",
+        "Year",
+        "Month",
+        "StateFIPS",
+        "StateName",
+        "CountyFIPS",
+        "state_abbrev",
+        "FIP_unique",
+        "Population",
+        "county_test",
+    ]
+]
+
+# creating a copy of reduced dataset of prescriptions and converting some of the attributes to appropriate data type
+
+prescriptions_reduced_copy = prescriptions_reduced.copy()
+
+prescriptions_reduced_copy["Year"] = prescriptions_reduced_copy["Year"].astype("int64")
+prescriptions_reduced_copy["DRUG_CODE"] = prescriptions_reduced_copy[
+    "DRUG_CODE"
+].astype("int64")
+prescriptions_reduced_copy["Month"] = prescriptions_reduced_copy["Month"].astype(
+    "int64"
+)
+prescriptions_reduced_copy["Population"] = prescriptions_reduced_copy[
+    "Population"
+].astype("int64")
+
+# creating a dataset that has all the drug prescriptions in the state of Florida
+
+washington_prescriptions = prescriptions_reduced_copy[
+    prescriptions_reduced_copy["StateName"] == "Washington"
+]
+# washington_prescriptions_copy = washington_prescriptions.copy()
```

Comment:
Don't comment this line out. It is better to create a copy of the washington_prescriptions subset, since you are adding a new column after line 62. You don't want to encounter SettingWithCopy Warning.